File: framework/languages/java/samples/java/DeliveryDispatch/Server/Configuration/src/main/java/systems/zlink/samples/deliverydispatch/server/configuration/SampleTimings.java
package systems.zlink.samples.deliverydispatch.server.configuration;

import java.time.Duration;

public final class SampleTimings {
    public static final Duration RequestTimeout = Duration.ofSeconds(10);
    public static final Duration OfferRequestTimeout = Duration.ofSeconds(2);
    public static final Duration CourierDecisionTimeout = Duration.ofMillis(900);

    private SampleTimings() {
    }
}

File: framework/languages/java/samples/kotlin/DeliveryDispatch/Server/Configuration/src/main/kotlin/systems/zlink/samples/kotlin/deliverydispatch/server/configuration/SampleTimings.kt
package systems.zlink.samples.kotlin.deliverydispatch.server.configuration

import java.time.Duration

object SampleTimings {
    val RequestTimeout: Duration = Duration.ofSeconds(10)
    val OfferRequestTimeout: Duration = Duration.ofSeconds(2)
    val CourierDecisionTimeout: Duration = Duration.ofMillis(900)
}
